src/app/components/PageLayoutWrapper/index.tsx

```diff
@@ -114,7 +123,7 @@ const PageLayoutWrapper = ({
       <ThemeProvider service={service} variant={variant}>
         {!isNextJs && <ServiceWorkerContainer />}
         <ManifestContainer />
-        <WebVitals pageType={pageType} />
+        {!isErrorPage && pageType && <WebVitals pageType={pageType} />}
```

Comment:
I think we need to remove the check for isErrorPage here, otherwise it won't register our `WS-ERROR-PAGE` in the web vitals dashboard...

OR have we made the decision that we no longer need web vitals on error pages?

```suggestion
        {pageType && <WebVitals pageType={pageType} />}
```

Reply:
@eagerterrier thinks we do not need to report on error pages.

Reply:
I would agree. Let's remove the logic which sets the page type, and keep this check here

Reply:
I have removed the logic that sets WS-ERROR-PAGE